libwacom/libwacom.h

```diff
@@ -79,24 +79,54 @@
  */
 
 /**
- @file libwacom.h
+ * @defgroup context libwacom context
+ * Functions to create and manage libwacom context.
+ *
+ * @defgroup devices libwacom devices
+ * Functions to create and manage libwacom devices.
+ *
+ * @defgroup styli libwacom styli
+ * Functions to create and manage libwacom styli.
  */
 
+/**
+ * @ingroup devices
+ */
 typedef struct _WacomDevice WacomDevice;
 
+/**
+ * @ingroup devices
+ */
 typedef struct _WacomMatch WacomMatch;
 
+/**
+ * @ingroup styli
+ */
 typedef struct _WacomStylus WacomStylus;
 
+/**
+ * @ingroup context
+ */
 typedef struct _WacomError WacomError;
 
+/**
+ * @ingroup context
+ */
 typedef struct _WacomDeviceDatabase WacomDeviceDatabase;
 
+/**
+ * @ingroup devices
+ */
 #define WACOM_STYLUS_FALLBACK_ID 0xfffff
+/**
+ * @ingroup devices
+ */
 #define WACOM_ERASER_FALLBACK_ID 0xffffe
```

Comment:
I would have expected these to be `@ingroup styli`

Reply:
indeed, thanks. fixed